services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/dsc/errors/CloudDscFeignHttpClientProviderException.java

```diff
@@ -0,0 +1,8 @@
+package app.coronawarn.server.services.distribution.dgc.dsc.errors;
+
+public class CloudDscFeignHttpClientProviderException extends RuntimeException {
```

Comment:
do we really need another `Dsc` exception here?
isn't it sufficient to use: `CloudFeignHttpClientProviderException` ???

Reply:
CloudFeignHttpClientProviderException is used for federation client. To be honest I will keep this one as these are different clients. Up to you :)